app/models/dhs1171_pdf.rb

```diff
@@ -0,0 +1,38 @@
+class Dhs1171Pdf
+  FIELDS = [
+    "applying_for_food_assistance",
+    "birth_day",
+    "birth_month",
+    "birth_year",
+    "city",
+    "county",
+    "full_name",
+    "signature",
+    "signature_date",
+    "state",
+    "street_address",
+    "zip",
+  ].freeze
+
+  def initialize(client_info)
+    @source_pdf = "DHS_1171.pdf"
+    @client_info = client_info
+  end
+
+  def save(new_pdf)
+    check_for_invalid_fields
+    PdfForms.new.fill_form(source_pdf, new_pdf, client_info)
+  end
+
+  private
+
+  attr_reader :client_info, :source_pdf
+
+  def check_for_invalid_fields
+    raise "Invalid fields passed in: #{invalid_client_fields}" if invalid_client_fields.any?
```

Comment:
Very minor style preference: I never see trailing conditionals, somehow, when I am reading and prefer block conditionals, but I'm a 2/10 on this position. Is this enforced by Rubocop?